SimulationAgent/Runtime/Config.cs

```diff
@@ -46,13 +53,23 @@ public Config(IConfigData configData)
                                     "value in the 'appsettings.ini' configuration file.");
             }
 
+            var limitsConf = new RateLimitingConfiguration
+            {
+                ConnectionsPerSecond = configData.GetDouble(CONNECTIONS_FREQUENCY_LIMIT_KEY),
```

Comment:
super nit: integer for connections

Reply:
I rather not, many values are expressed as real numbers also in our docs, I consider this as a feature (e.g. 2.5 connections per second is a possible scenario)

Reply:
From a hub side only integer works.  I can see a world where you are going so far as to fine tune as you're suggesting.  I'll be surprised though :).

Reply:
changed to Int in the other PR